Sources/SectionScrubber.swift

```diff
@@ -20,14 +26,14 @@ public class SectionScrubber: UIView {
 
     public var delegate : SectionScrubberDelegate?
 
-    public var viewHeight : CGFloat = 56.0
-
-    public var scrubberWidth : CGFloat = 22.0
-
     public var containingViewFrame = UIScreen.mainScreen().bounds
 
     public var containingViewContentSize = UIScreen.mainScreen().bounds.size
 
+    public var viewHeight : CGFloat = 56.0
```

Comment:
I think it would be more consistent if we used the notation 

```
public var viewHeight = CGFloat(56.0)
```

Since we are not doing `var varName: TypeName` anywhere else.